pkg/io/binaryBufWriter.go

```diff
@@ -19,6 +19,11 @@ func NewBufBinWriter() *BufBinWriter {
 	return &BufBinWriter{BinWriter: NewBinWriterFromIO(b), buf: b}
 }
 
+// Len returns the number of bytes of the unread portion of the buffer.
+func (bw *BufBinWriter) Len() int {
```

Comment:
I don't see it being used, do we really need it?

Reply:
in codegen.go:
```
// pc returns the program offset off the last instruction.
func (c *codegen) pc() int {
	return c.prog.Len() - 1
}
```

Reply:
Ah, it's not changed in this patchset, so it's kinda invisible. ACK.